presto-main/src/main/java/io/prestosql/metadata/Metadata.java

```diff
@@ -379,6 +380,12 @@
 
     TypeManager getTypeManager();
 
+    //
```

Comment:
This is misleading since there's only two block functions, followed by more non-block methods. Probably move these to the end.